tokio/src/executor/global.rs

```diff
@@ -15,7 +15,7 @@ use std::pin::Pin;
 /// executor (usually itself) that is used to spawn new tasks.
 ///
 /// The current `DefaultExecutor` reference is tracked using a thread-local
-/// variable and is set using `tokio_executor::with_default`
+/// variable and is set using `crate::executor::with_default`
```

Comment:
This looks wrong. Looks like a str-replace error, so probably happened elsewhere too.